sdk/formrecognizer/Azure.AI.FormRecognizer/src/ClientCommon.cs

```diff
@@ -88,5 +89,35 @@ public static RecognizedFormCollection ConvertPrebuiltOutputToRecognizedForms(V2
             }
             return new RecognizedFormCollection(forms);
         }
+
+        public static IReadOnlyList<DocumentWord> GetWords(IReadOnlyList<DocumentPage> pages, IReadOnlyList<DocumentSpan> spans)
```

Comment:
Can you make this `IEnumerable` instead of `IReadOnlyList`?  Doesn't look like you're indexing anything in those lists.